proposals/rcp-035-single-feeds.md

```diff
@@ -0,0 +1,141 @@
+# RESO (Endorsement Name) Endorsement
+
+| **RCP** | 035 |
+| :--- | :--- |
+| **Version** | **DD 2.1** |
+| **Authors** | Eric Finlay |
+| **Specification** | [RCP-035](https://github.com/RESOStandards/transport/issues/96) |
+| **Status** | IN PROGRESS |
+| **Date Ratified** | Pending |
+| **Dependencies** | None |
+| **Related Links** | [Single Feed Discussion](https://github.com/RESOStandards/transport/discussions/71)|
+
+
+<br /><br />
+
+# RESO End User License Agreement (EULA)
+
+This End User License Agreement (the "EULA") is entered into by and between the Real Estate Standards Organization ("RESO") and the person or entity ("End User") that is downloading or otherwise obtaining the product associated with this EULA ("RESO Product"). This EULA governs End Users use of the RESO Product and End User agrees to the terms of this EULA by downloading or otherwise obtaining or using the RESO Product.
+
+<br /><br />
+
+# Table of Contents
+- [Summary of Changes](#summary-of-changes)
+- [Introduction](#introduction)
+- [Section 1: Purpose](#section-1-purpose)
+- [Section 2: Specification](#section-2-specification)
+- [Section 3: Certification](#section-3-certification)
+- [Section 4: Contributors](#section-4-contributors)
+- [Section 5: References](#section-5-references)
+- [Section 6: Appendices](#section-6-appendices)
+- [Section 7: License](#section-7-license)
+
+<br /><br />
+
+# Summary of Changes
+
+A new field called `FeedTypes` of type Open Mutli Enum String will be included in every data resource and the `Fields` metadata resource.
```

Comment:
Thanks for changing to `FeedTypes`

Should be `Field` metadata resource or `Field Resource`.

Reply:
Done